ui/index.html

```diff
@@ -23,7 +23,7 @@
 
         <!-- BODY -->
         <main id="index-container" class="oae-branding-container oae-main-content" role="main">
-            <div id="index-page-container" class="row"><!-- --></div>
+            <div id="index-page-container" tabindex="0" class="row"><!-- --></div>
```

Comment:
Is it common practice to have a div be the target for a skip to content link? Wouldn't it be better to link to an element that would provide some content (e.g. header that says `Content begins here`) to the screenreader user (and would be natively focusable) ?